actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Deprecate `assigns` and `assert_template`.
```

Comment:
I think it is better to tell that these methods were extracted to the gem and point to it on the changelog.